pkg/util/api/api.go

```diff
@@ -38,3 +44,14 @@ func truncateMessage(message string, limit int) string {
 	suffix := " ..."
 	return message[:limit-len(suffix)] + suffix
 }
+
+// ObjectMetaForCreation creates a copy of the provided ObjectMeta containing
+// only the name, namespace, labels and annotations
+func ObjectMetaForCreation(orig *metav1.ObjectMeta) metav1.ObjectMeta {
```

Comment:
I doubt this function is general enough to be in the api. Maybe we can place it closer to the implementation, for example in some utils for multikueue, or job adapters?

Reply:
In my opinion it fits best here, creating an util package just for this is too much in my opinion, and the job frameworks is not its place.

Reply:
ok, though it would be good to have a util package dedicated for that, since this is only used in multikueue adapters.

Reply:
We can think of that if we get more multikueue specific functions.

Reply:
```suggestion
func CloneObjectMetaForCreation(orig *metav1.ObjectMeta) metav1.ObjectMeta {
```

Reply:
done